src/commands/convert/sarif2hdf.ts

```diff
@@ -4,16 +4,17 @@ import {SarifMapper as Mapper} from '@mitre/hdf-converters'
 import {checkInput, checkSuffix} from '../../utils/global'
 
 export default class Sarif2HDF extends Command {
-  static usage = 'convert sarif2hdf -i, --input=JSON -o, --output=OUTPUT'
+  static usage = 'convert sarif2hdf -i, --input=JSON -o, --output=OUTPUT -w, --with-raw'
 
   static description = 'Translate a SARIF JSON file into a Heimdall Data Format JSON file\nSARIF level to HDF impact Mapping:\nSARIF level error -> HDF impact 0.7\nSARIF level warning -> HDF impact 0.5\nSARIF level note -> HDF impact 0.3\nSARIF level none -> HDF impact 0.1\nSARIF level not provided -> HDF impact 0.1 as default'
 
-  static examples = ['saf convert sarif2hdf -i sarif-results.json -o output-hdf-name.json']
+  static examples = ['saf convert sarif2hdf -i sarif-results.json -o output-hdf-name.json -w']
 
   static flags = {
     help: Flags.help({char: 'h'}),
-    input: Flags.string({char: 'i', required: true}),
-    output: Flags.string({char: 'o', required: true}),
+    input: Flags.string({char: 'i', required: true, description: 'Input SARIF file'}),
+    output: Flags.string({char: 'o', required: true, description: 'Output HDF file'}),
```

Comment:
These descriptions are not consistent with the README (but https://github.com/mitre/saf/pull/526 solves this)